script/release

```diff
@@ -62,13 +54,14 @@ puts
 puts "From: #{original_version}"
 puts "To:   #{version_string}"
 puts
+
 puts "Type 'yes' to continue"
 x = STDIN.gets
 exit(1) unless x.strip == 'yes'
 puts
 
 verbose('git fetch origin')
-verbose('git reset --hard origin/master')
+verbose('git reset --hard origin/master') if release_option != "5"
 verbose('git clean -ffxd .')
 
 File.write(changelog_file, File.read(changelog_file).gsub(/\[Unreleased\]/, "[#{version_string}] - #{Date.today}"))
```

Comment:
Since we only have one commit on top of the last publish tagged commit, we shouldnt have any changelog items here right?

Reply:
We do. We will have the hotfix commit before this. This is the release commit on top of the hotfix.